packages/cdc-react/src/components/ProfileHeader/ProfileHeaderUserProfileMenuItem/ProfileHeaderUserProfileMenuItem.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { PopupMenuItemType } from "../../../@types";
+
+import { render, screen } from "@testing-library/react";
+// import userEvent from "@testing-library/user-event";
+
+import { ProfileHeaderUserProfileMenuItem } from "./ProfileHeaderUserProfileMenuItem";
+
+describe("ProfileHeaderUserProfileMenuItem component", () => {
+  const popupMenuItems: PopupMenuItemType[] = [
+    {
+      icon: "user",
+      iconPosition: "left",
+      text: "Your Profile",
+      onClick: undefined,
+      badgeCount: 0,
+    },
+    {
+      icon: "notifications",
+      iconPosition: "left",
+      text: "Notifications",
+      onClick: undefined,
+      badgeCount: 1,
+    },
+    {
+      icon: "settings",
+      iconPosition: "left",
+      text: "Settings",
+      onClick: undefined,
+      badgeCount: 0,
+    },
+    {
+      icon: "logout",
+      iconPosition: "left",
+      text: "Logout",
+      onClick: undefined,
+      badgeCount: 0,
+    },
+  ];
+
+  it("should render the popup menu", () => {
```

Comment:
This test seems invalid to me.  It should only render the menu after the user profile button has been clicked.  Otherwise, the menu items should not be in the document.

Reply:
Changing it from a class should resolve this.